manifests/03-role.yaml

```diff
@@ -78,6 +78,12 @@ rules:
   - "privileged"
   verbs:
   - use
+- apiGroups:
```

Comment:
This has to be added to:
* https://github.com/openshift/cluster-logging-operator/blob/master/controller-manifests/cluster-logging.v4.1.0.clusterserviceversion.yaml
* https://github.com/operator-framework/community-operators/blob/master/community-operators/cluster-logging/cluster-logging.v4.1.0.clusterserviceversion.yaml

Reply:
ad 1 - in this PR
ad 2 - merged already https://github.com/operator-framework/community-operators/commit/d7cf566847540e981e358b23d30ef5ecfd627b9f